Readme.md

```diff
@@ -42,42 +96,56 @@ Format the given value in bytes into a string. If the value is negative, it is k
 **Example**
 
 ```js
-bytes(1024);
+bytes(1000);
 // output: '1kB'
 
 bytes(1000);
 // output: '1000B'
 
-bytes(1000, {thousandsSeparator: ' '});
-// output: '1 000B'
+bytes(1000, {thousandsSeparator: ','});
+// output: '1,000B'
+
+bytes(1024);
+// output: '1.02kB'
 
-bytes(1024 * 1.7, {decimalPlaces: 0});
+bytes(2078, {decimalPlaces: 0});
```

Comment:
So this example was to demo the rounding. Show that 1024 times a value between 1 and 2 is more obvious I would say to understand that the value gets rounded, where as a single number is really hard to understand that same thing.